lib/vacations.ex

```diff
@@ -0,0 +1,143 @@
+defmodule GlassFactoryApi.Vacations do
+  @moduledoc """
+  Provides the methods to get vacations of all or a specific user.
+  """
+
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Vacations.Vacation
+
+  @doc """
+  Return a tuple with `:ok` atom and an list of vacations.
+  or an tuple with `:error` and a string with the error description.
+
+  It's possible apply some filters, like user_id, start and end to limit dates, and vacation_type
+  using the query_string param.
+
+  ## Examples
+
+  iex> GlassFactoryApi.Clients.get_vacations()
```

Comment:
I think we need to indent this with 4 space in order to render properly in ex_doc.

Reply:
Just ran `ex_doc` and confirmed: we do need to indent this.